src/gatling/conf/gatling.conf

```diff
@@ -24,22 +25,53 @@ gatling {
       }
       jsonPath {
         #cacheMaxCapacity = 200 # Cache size for the compiled jsonPath queries, set to 0 to disable caching
-        #preferJackson = false  # When set to true, prefer Jackson over Boon for JSON-related operations
       }
       css {
         #cacheMaxCapacity = 200 # Cache size for the compiled CSS selectors queries,  set to 0 to disable caching
       }
     }
-
     directory {
-      #data = user-files/data               # Folder where user's data (e.g. files used by Feeders) is located
-      #bodies = user-files/bodies           # Folder where bodies are located
-      simulations = src/gatling/simulations/simulations # Folder where the bundle's simulations are located
+      #simulations = user-files/simulations # Directory where simulation classes are located (for bundle packaging only)
```

Comment:
Please replace this file with the version for 3.9.5 from the template-performance repo

Reply:
Changed